src/ServicePulse.Host/vue/src/components/configuration/RetryRedirects.vue

```diff
@@ -71,7 +71,7 @@ async function saveEditedRedirect(redirect) {
     getRedirect();
   } else {
     redirectSaveSuccessful.value = false;
-    if (result.status === "409" || result.status === 409) {
```

Comment:
I'm guessing this is TypeScript telling us this can only be a number

Reply:
yes I typed the `status` to be a `number`